templates/components/_nav_mobile.html.twig

```diff
@@ -0,0 +1,212 @@
+    </ul>
+    <div class="nav-mobile">
+        <input type="checkbox" id="nav-menu">
+        <label for="nav-menu">Menu de navigation</label>
+        <div class="nav-mobile--menu">
+            {% if (not app.user) %}
+                <ul>
+                    <li>
+                        <a href="{{ path('app_user_login') }}">Connexion</a>
+                    </li>
+                    <li>
+                        <a href="{{ path('app_membership_register') }}">Inscription</a>
+                    </li>
+                </ul>
+                <hr />
+            {% endif %}
+            {% if has_role_user %}
+                <ul>
+                    <li>
+                        <a href="{{ has_role_adherent ? path('app_user_profile') : path('app_user_login') }}">
+                            <div class="text--bold">{{ app.user.firstName }}</div>
+                            <div class="text--small text--gray">
+                                {% if not has_role_adherent %}
+                                    Non-adhérent(e)
+                                {% elseif is_granted('ROLE_REFERENT') %}
+                                    {% if app.user.isFemale %}Référente 🥇{% else %}Référent 🥇{% endif %}
+                                {% elseif app.user.isHost %}
+                                    {% if app.user.isFemale %}Animatrice 🏅{% else %}Animateur 🏅{% endif %}
+                                {% elseif has_role_adherent %}
+                                    {% if app.user.isFemale %}Adhérente 😍{% else %}Adhérent 😍{% endif %}
+                                {% endif %}
+                            </div>
+                        </a>
+                    </li>
+                </ul>
+                <hr />
+                <ul>
+                    <li>
+                        {% set url = has_role_adherent ? path('app_user_profile') : path('app_user_login') %}
+                        <a href="{{ url }}">Profil</a>
+                    </li>
+                    <li>
+                        <a href="{{ path('app_adherent_events') }}">Mes activités</a>
+                    </li>
+                </ul>
+                {% if is_granted('CREATE_CITIZEN_PROJECT') %}
+                    <ul>
+                        <li>
+                            <a href="{{ path('app_adherent_create_citizen_project') }}">Lancer mon projet</a>
+                        </li>
+                    </ul>
+                {% endif %}
+                <ul>
+                    {% if has_role_adherent and is_granted('CREATE_COMMITTEE') %}
+                        <ul>
+                            <li>
+                                <a href="{{ path('app_adherent_create_committee') }}">Créer un comité</a>
+                            </li>
+                        </ul>
+                    {% endif %}
+                    {% if is_granted('ROLE_REFERENT') %}
```

Comment:
Same is here and below, use `app.user.*` when possible

Reply:
fixed